packaging/linux/deb/openra.lintian-overrides

```diff
@@ -0,0 +1,3 @@
+openra: duplicate-font-file usr/lib/openra/mods/common/FreeSans.ttf also in fonts-freefont-ttf
+openra: duplicate-font-file usr/lib/openra/mods/common/FreeSansBold.ttf also in fonts-freefont-ttf
+openra: binary-without-manpage usr/games/openra-server
```

Comment:
Well, this is a thirdparty not a distribution package so duplicating is better, but why do you turn off (or what else does override mean) this warnings?